coal_petroleum/data/res_config_settings.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
```

Comment:
We should use data noupdate="1" because of the function here

Reply:
Ok, then enclose the record with `<data noupdate="1">`. We don't want to re-activate settings if they unactivate them.
```suggestion
<odoo>
    <data noupdate="1">
        <record id="res_config_settings_multi_step_routes" model="res.config.settings">
            ...
        </record>
    </data>
    
    <function model="res.config.settings" name="execute">
        ...
```

Reply:
In fact, all industries have a res_config_settings starting with `<odoo noupdate="1">`. There is no point in calling the function at the update, since no modification is done to the settings.